### ISSUE TITLE
Completion suggested in wrong places

### ISSUE THREAD

**Issue by MontakOleg** [bealex/SwiftLintAppCode#60]
Hello!

<img width="546" alt="изображение" src="https://user-images.githubusercontent.com/1800899/153411912-4fdf4eee-a250-401d-8e3d-14c14a0a0609.png">

If I understand intent correctly (restrict completion on /// comments), this change is not correct:

https://github.com/bealex/SwiftLintAppCode/commit/2dc2259d3d25a062db61d4dd13f39d851ac2a5d6#diff-ca6c0092682e7239f9a53dad84c7e48c5fc3c77de30421dc589f6f12985a70daR83-R84

I think it should be something like this:
```
if (position.parent.elementType !is SwiftLazyEolCommentElementType || position.parent.firstChild.text != "//") return
```

**Comment by bealex** [comment]
Hmm. You're absolutely right! And I'm dumb, as usual :)

**Comment by bealex** [comment]
Should be fixed in 1.13.6
